src/include/daos/common.h

```diff
@@ -101,6 +101,27 @@ daos_u32_hash(uint64_t key, unsigned int bits)
 
 #define LOWEST_BIT_SET(x)       ((x) & ~((x) - 1))
 
+static inline uint8_t
+isset_range(uint8_t *bitmap, uint8_t start, uint8_t end)
+{
+	int index;
+
+	for(index = start; index <= end; ++index)
+		if(isclr(bitmap,index))
```

Comment:
(style)  space required after that ',' (ctx:VxV)
(style)  space required before the open parenthesis '('